using_images/s2i_images/customizing_s2i_images.adoc

```diff
@@ -36,7 +36,8 @@ To determine the location of the scripts inside of the builder image, look at
 at the value of `*io.openshift.s2i.scripts-url*` label. Use `docker inspect`:
 
 ----
-$ docker inspect --format='{{ index .Config.Labels "io.openshift.s2i.scripts-url" }}' openshift/wildfly-100-centos7
+$ docker inspect --format='{{ index .Config.Labels "io.openshift.s2i.scripts-url" }}' /
```

Comment:
It should be a backslash (not a forward slash) to indicate a line break, but I can remove it.